packages/python-packages/protocol-stub-generator/setup.py

```diff
@@ -0,0 +1,48 @@
+from setuptools import setup, find_packages
+import os, re
+
+PACKAGE_NAME = "protocol-stub-generator"
+
+DESCRIPTION = (
+    "A llc stub generator for published APIs, variables and properties in a package"
+)
+
+with open(os.path.join("protocol", "_version.py"), "r") as fd:
+    version = re.search(
+        r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]', fd.read(), re.MULTILINE
+    ).group(1)
+
+if not version:
+    raise RuntimeError("Cannot find version information")
+
+with open("README.md", encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    description=DESCRIPTION,
+    version=version,
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Azure/azure-sdk-tools/",
+    author="Microsoft Corporation",
+    author_email="azuresdkengsysadmins@microsoft.com",
+    license="MIT License",
+    packages=find_packages(),
+    install_requires=["astroid"],
+    python_requires=">=3.4.0",
+    entry_points={
+        "console_scripts": [
+            "protocol_api=protocol:console_entry_point",
+        ]
+    },
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```

Comment:
not super important since we won't actually publish this to PyPi, but we can remove python 3.4 here and add 3.8 and 3.9